components/Card.js

```diff
@@ -2,9 +2,15 @@ import styled from "styled-components";
 import Image from "next/image";
 import deleteIcon from "../public/images/deleteIcon.svg";
 
-export default function Card({ name, onDelete }) {
+export default function Card({ id, name, onDelete, onToggle, isChecked }) {
   return (
-    <CardContainer>
+    <CardContainer isChecked={isChecked}>
+      <StyledInput
+        id={id}
+        type="radio"
+        defaultChecked={isChecked}
+        onClick={onToggle}
+      />
       {name}
```

Comment:
put the name in a `<label>` element and connect it with the input via `htmlFor`.. then you can also toggle your habit by clicking the name, not only the checkbox ;)

Reply:
Thank you for your great suggestion. i inserted the <labe>